packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -12,6 +12,16 @@ import type { WalletConnectOptions } from '@web3-onboard/walletconnect/dist/type
 import type { AppMetadata } from '@web3-onboard/common';
 import { utils } from 'ethers';
 import { WalletUpdateEvent, walletContext } from '../../context';
+import type { Eip1193Provider } from '../../interfaces';
+
+type ChainData = {
```

Comment:
i think adding a comment here would be better to provide context of the source list